accessible/tree.service.js

```diff
@@ -484,8 +484,8 @@ blocklyApp.TreeService = ng.core.Class({
         // Return the focus to the workspace tree containing the input field.
         document.getElementById(treeId).focus();
 
-        // Note that Tab events are allowed to propagate through.
-        if (e.keyCode == 27) {
+        // Note that Tab and Enter events are allowed to propagate through.
```

Comment:
Not your comment, but do you know why it says "are allowed to propagate" when the code is calling stopPropagation?